City-of-Helsinki/kukkuu-ui
# KK-937 | Replace Hurjaruuth's logo with a new one

## Description

Replace Tanssiteatteri [Hurjaruuth](https://www.hurjaruuth.fi/)'s logo with a new one.

Reduced image size and optimized from original 2080x1520 to 500x365.

Also optimize the biggest logo file tanssintalo.png without changing its resolution.

## Context

[KK-937](https://helsinkisolutionoffice.atlassian.net/browse/KK-937)

## How Has This Been Tested?

Tested locally by opening the user interface's frontpage and saw that Hurjaruuth's logo had changed.

## Manual Testing Instructions for Reviewers

Open the user interface's frontpage and look at the logos near the bottom of the page,
see the Hurjaruuth's logo.

## Screenshots

### Before
![before](https://user-images.githubusercontent.com/77663720/187228933-7d634bed-0c9e-4203-9fd6-3fc1ad9e42bc.png)

### After
![after](https://user-images.githubusercontent.com/77663720/187228943-5a497883-028d-4a2e-a87c-7f210717e0ee.png)
